packages/terra-card/src/CardPadding.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import styles from './Card.scss';
+
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Child Nodes
+   */
+  children: PropTypes.node,
+  /**
+   * Provides 10px of themeable padding vertical
+   */
+  hasPaddingVertical: PropTypes.bool,
+  /**
+   * Provides 10px of themeable padding horizontal
+   */
+  hasPaddingHorizontal: PropTypes.bool,
+};
+
+const defaultProps = {
+  hasPaddingHorizontal: false,
+  hasPaddingVertical: false,
+};
+
+
+const CardPadding = ({
+  children,
+  hasPaddingVertical,
+  hasPaddingHorizontal,
+  ...customProps
+}) => {
+  const cardPaddingClasses = cx([
+    { 'veritcal-padding': hasPaddingVertical },
+    { 'horizontal-padding': hasPaddingHorizontal },
+    customProps.className,
+  ]);
+  return <div {...customProps} className={cardPaddingClasses}>{children}</div>;
```

Comment:
Previously in the war room we discussed padding options to either always inject a div (like this) or to instead to use custom props to add padding to the child component. I think there are pros and cons to both approaches. I'd like to have us come to a decision on which method we prefer before we add this component.